File: QA/Tower_ODR/Tower_ODR/Tower_ODR/InboundDocumentMappingTests.cs
using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace Tower_ODR
{
    [TestClass]
    public class InboundDocumentMappingTests
    {

        [TestMethod]
        public void InboundDocumentMappingTestsLoadIsSuccessfull()
        {
            TowerAutomationTests.TestProtractormethod("InboundDocumentMappingTests", "Verify InboundDocumentMapping Screen Loaded properly");
        }

        [TestMethod]
        public void InboundDocumentMappingAddDeleteDocMapping()
        {
            TowerAutomationTests.TestProtractormethod("InboundDocumentMappingTests", "Verify Add and delete inbound document mapping");
        }


    }
}

File: QA/EndToEndTesting/TestSuites/InboundDocumentMappingTests.js
describe('Test Execution on InboundDocumentMapping', function () {

    'use strict';
        
    var pages = {};
    var testData = require('../resources/testData.json');
    pages.InboundDocumentMapping = require('../pages/InboundDocumentMapping.js');
    pages.Customers = require('../pages/Customers.js');

    var inboundDocumentMappingPage = new pages.InboundDocumentMapping();
    var customerPage = new pages.Customers();
    
    
    it('Verify InboundDocumentMapping Screen Loaded properly', function () {
        inboundDocumentMappingPage.openSearchPage(testData.search.homeUrl);
        inboundDocumentMappingPage.isPageLoaded();     
    });

    it('Verify Add and delete inbound document mapping', function(){
        inboundDocumentMappingPage.addanddeleteNewInboundDocument();
    });

});